README.md

```diff
@@ -22,6 +22,18 @@ rm .install-filcrypto \
     ; make clean \
     ; FFI_BUILD_FROM_SOURCE=1 make
 ```
+## go get
+
+`go get` needs some additional steps in order to work as expected.
+
+Get the source, add this repo as a submodule to your repo, build it and point to it:
+
+```shell
+$ go get github.com/filecoin-project/filecoin-ffi
+$ git submodule add https://github.com/filecoin-project/filecoin-ffi.git extern/filecoin-ffi
+$ make -C extern/filecoin-ffi
+$ echo 'replace github.com/filecoin-project/filecoin-ffi => ./extern/lotus/extern/filecoin-ffi' >> go.mod
```

Comment:
if we want to do that as command, better do:
`go mod edit -replace=github.com/filecoin-project/filecoin-ffi=./extern/filecoin-ffi`

Reply:
TIL that `go mod edit -replace` was a thing